config.md

```diff
@@ -308,15 +308,23 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps where both the key and value are strings.
+Keys MUST be unique, and best practice is to namespace the keys.
+Keys starting with the `opencontainers.org` namespace are reserved and MUST NOT be used.
+If there are no annotations then this property MAY either be absent or an empty map.
+Implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown annotation key.
```

Comment:
I don't think we can mandate they ignore them because that's how people may turn on/off extensions.